config/clades-short_h1n1pdm_ha.tsv

```diff
@@ -0,0 +1,89 @@
+clade	gene	site	alt
+2	HA1	31	D
+2	HA1	162	N
+2	HA1	186	T
+2	nuc	1100	A
+3	HA1	183	P
+3	HA1	134	T
+3	nuc	1484	T
+4	HA1	125	D
+4	nuc	1301	A
+4	HA2	47	K
+4	HA1	283	K
+6	HA1	97	N
+6	HA2	124	N
+6	HA1	185	T
+7	HA1	143	G
+7	nuc	164	T
+7	nuc	1457	T
+7	nuc	1676	T
+8	SigPep	15	T
+8	HA1	186	T
+8	HA1	272	A
+8	HA2	146	D
+8	HA2	147	K
+6B	HA1	163	Q
+6B	HA1	256	T
+6B	nuc	1673	C
+6C	HA1	234	I20

```

Comment:
Just noticed this typo while I was renaming the files. I will remove this in a subsequent commit.